hastexo_guacamole_client/settings.py

```diff
@@ -1,24 +1,19 @@
 import os
+from yaml import load, SafeLoader
+from yaml.scanner import ScannerError
 
 from django.core.exceptions import ImproperlyConfigured
 
 
-def get_env_setting(setting):
-    """ Get the environment setting or return exception """
-    try:
-        return os.environ[setting]
-    except KeyError:
-        error_msg = u"Set the %s env variable" % setting
-        raise ImproperlyConfigured(error_msg)
-
-
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = get_env_setting('HASTEXO_GUACAMOLE_SECRET_KEY')
+SECRET_KEY = os.environ.get('HASTEXO_GUACAMOLE_SECRET_KEY',
+                            'insecure-secret-key')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = os.getenv('HASTEXO_GUACAMOLE_DEBUG', False)
+DEBUG = os.environ.get('HASTEXO_GUACAMOLE_DEBUG', False)
```

Comment:
I believe you need to wrap this in `bool(strtobool())`, otherwise setting the environment variable to the string `false` will result in `DEBUG = True`.

Reply:
done.